src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -1103,7 +1055,15 @@ public bool ProcessTransaction(Transaction transaction)
             lock (this.lockProcess)
             {
                 foreach (string walletName in this.WalletRepository.GetWalletNames())
+                {
+                    this.logger.LogDebug("Processing transaction '{0}' for wallet {1}.", transaction.GetHash(), walletName);
+                    foreach (var input in transaction.Inputs)
```

Comment:
This loop does not need to be done for each wallet.

Reply:
👍